backend/src/components/rfiApplication.js

```diff
@@ -95,51 +100,67 @@ async function updateRFIApplication(req, res) {
     let friApplicationResponse = await patchOperationWithObjectId('ccof_rfipfis', rfipfiid, friApplication);
     friApplicationResponse = new MappableObjectForFront(friApplicationResponse, RFIApplicationMappings);
 
-    // update isComplete status
```

Comment:
is this a merge issue.  not sure why this is being removed.

Reply:
I don't think it's being removed, I think the code is just being shifted down?

Reply:
Yah I think it was a merge issue - I don't recall taking this out. I've added it back in